src/pages/module-deactivation/index.js

```diff
@@ -160,11 +160,7 @@ const ModuleDeactivation = () => {
           margin: 0
         }}
       >
-        <WindowToolbar 
-          isSaved={true}
-          onSave={handleSubmit} 
-          smallBox={true} 
-        />
+        <WindowToolbar isSaved={true} onSave={handleSubmit} smallBox={true} />
```

Comment:
the only way here is to define table then window with save button ? is it possible to use inside form shell table (like what previously i have seen in datagrid)?

Reply:
ang what is the importance of defining module store above? i know that we should prevent using states as much as possible

Reply:
moduleState used here since it's required to store data returned from getAllKvsByDataset